templates/ai/services.html

```diff
@@ -7,316 +7,306 @@
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+      </div>
+      <p>
+        <a class="p-link--external" href="https://canonical.com/partners/become-a-partner"><small>Become a trusted partner</small></a>
+      </p>
     </div>
-    <div class="col-6">
-      <h4>3 - Design</h4>
+    <div class="col-8">
+      <h2>Enterprise MLOps training</h2>
+      <p class="p-heading--4">4&ndash;days to success with machine learning on Kubeflow</p>
+      <p>Grow end-to-end machine learning knowledge within your team, with enterprise training by Canonical.</p>
+      <p>Align your sysadmins, devops, data engineers and data scientists with the latest MLOps solutions.</p>
+      <p>Any infra and level of expertise. Tailored to your use cases. Up to 15 attendees. 4-day on-premise training for $29.500.</p>
       <p>
-        Prototype new solutions and identify a candidate solution that seems to be the best fit for the use case under consideration. Data engineers will prepare the data, data scientists will discuss and select appropriate features and machine learning algorithms, and machine Learning engineers will design, build and perform preliminary tests on your prototype neural network. Time permitting, the iterative process of design and discovery on the data and the neural network model will start.
+        <a class="p-link--external p-button--neutral" href="https://assets.ubuntu.com/v1/ee5a96f3-Kubeflow+Training+Curriculum.pdf?_ga=2.202291033.951646160.1607941648-154096706.1586788067">See Curriculum</a>
+        <a class="p-button--neutral js-invoke-modal" href="/training/contact-us?product=kubeflow-training-onsite">Talk to us</a>
       </p>
     </div>
   </div>
+</section>
 
-  <div class="u-fixed-width">
-    <hr class="p-separator"/>
-  </div>
-
-  <div class="row">
-    <div class="col-6">
-      <h4>4 - Implementation</h4>
-      <p>
-        Complete the design process and begin training and testing your AI model until it reaches the desired accuracy threshold. A kubeflow pipeline that will put your model into a suitable environment for testing and feedback from additional stakeholders will be built. Domain experts will offer guidance on assessing machine learning predictions and putting discovered insights into action.
-      </p>
+<section class="p-strip--square-darksuru is-dark">
+  <div class="row u-equal-height">
+    <div class="col-7 u-vertically-center">
+      <div>
+        <h2>Easy Kubeflow operations</h2>
+        <p>Charmed Kubeflow integrates the 30+ apps that make up Kubeflow with ops code to provide the best Kubeflow experience, from deployment to day-2 operations. </p>
+        <br />
+          <a class="p-link--external p-button--positive" href="https://charmed-kubeflow.io">Visit Charmed-Kubeflow.io</a>
+          <a class="p-link--external p-link--inverted" href="https://assets.ubuntu.com/v1/197d9867-Charmed_Kubeflow_Kubernetes_10.11.20+%282%29.pdf">Download datasheet</a>
```

Comment:
```suggestion
          <a class="p-link--inverted" href="https://assets.ubuntu.com/v1/197d9867-Charmed_Kubeflow_Kubernetes_10.11.20+%282%29.pdf">Download the datasheet</a>
```

again, not external, and we need definate articles